src/django/api/management/commands/export_csv.py

```diff
@@ -18,42 +18,72 @@ def print_with_time(text: str) -> None:
     print(f"{datetime.now().strftime('%Y-%m-%d %H:%M:%S')} -- {text}")
 
 
+def divide_chunks(list, size, amount):
+    chunks = []
+    last_index = size
+    first_index = 0
+    for i in range(1, amount + 1):
+        if i == amount:
+            chunks.append(list[first_index:])
+        else:
+            chunks.append(list[first_index:last_index])
+            first_index = last_index
+            last_index = last_index + size
+    return chunks
+
+
 class Command(BaseCommand):
     help = 'Export all facilities to a CSV file.'
 
     def add_arguments(self, parser):
         return
-        print("add_arguments")
 
     def handle(self, *args, **options):
-        print_with_time("Started indexing facilities")
-        index_facilities_new()
         facilities = FacilityIndex.objects.order_by("id").all()
         count = facilities.count()
-        print_with_time(f"Finished indexing {count} facilities")
+        size_of_chunk = 1000
+        amount_of_chunks = math.ceil(count / size_of_chunk)
+        print_with_time(f"Amount of indexed facilities: {count}")
+        print_with_time(f"Amount of chunks: {amount_of_chunks}")
 
-        serializer = FacilityIndexDownloadSerializer()
+        facilities_pool = divide_chunks(
```

Comment:
"Instead of getting all chunks at once, I suggest implementing lazy loading or streaming so that chunks are created and processed sequentially

Reply:
Yes, I think about it as a next step.